gen2-facemesh/main.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import cv2
+import depthai as dai
+import argparse
+import time
+import numpy as np
+
+'''
+YoloV5 object detector running on selected camera.
```

Comment:
Perhaps update this with info about Google MediaPipe, FaceMesh, and source of the converted model.

Reply:
Yeah, sorry. Was trying to get this up ASAP. Will update this later today. Thanks! :)

Reply:
Not a problem, I appreciate how quickly you made this :D